src/redux/selectors/siloDomainData/addLeads.js

```diff
@@ -130,16 +134,21 @@ export const addLeadViewLeadKeysSelector = createSelector(
 
 export const addLeadViewLeadStatesSelector = createSelector(
     // TODO: distribute this later
+    activeProjectRoleSelector,
     addLeadViewLeadRestsSelector,
     addLeadViewLeadUploadsSelector,
     addLeadViewLeadDriveRestsSelector,
     addLeadViewLeadDropboxRestsSelector,
     addLeadViewLeadsSelector,
-    (leadRests, leadUploads, leadDriveRests, leadDropboxRests, leads) => (
+    (projectRole, leadRests, leadUploads, leadDriveRests, leadDropboxRests, leads) => (
```

Comment:
Maybe consider it changing to one object instead of millions of arguments,
Use multiple lines

Reply:
it's legacy code now. breaking this to multiple lines will only make it more confusing